systest/integration2/graphql_schema_auth_test.go

```diff
@@ -0,0 +1,74 @@
+//go:build integration2
+
+/*
+ * Copyright 2023 Dgraph Labs, Inc. and Contributors *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"testing"
+	"time"
+
+	"github.com/dgraph-io/dgraph/dgraphtest"
+	"github.com/dgraph-io/dgraph/x"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGraphqlSchema(t *testing.T) {
+	conf := dgraphtest.NewClusterConfig().WithNumAlphas(2).WithNumZeros(1).
+		WithACL(time.Hour).WithReplicas(1).WithGraphqlLambdaURL("http://127.0.0.1/temp")
```

Comment:
Is `http://127.0.0.1/temp` a dummy url or there a Lambda server running at this location?

Reply:
dummy URL, added a comment. This PR is now good to go.